charts/argo-cd/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: v2.4.15
+appVersion: v2.5.0-rc3
```

Comment:
To be changed to 2.5.0 when released

Reply:
it has been done :smile: https://github.com/argoproj/argo-cd/releases/tag/v2.5.0

Reply:
@jmeridth Builiding